rest/data/interface.go

```diff
@@ -136,10 +140,23 @@ type BuildloggerOptions struct {
 // TestResultsOptions holds all values required to find a specific TestResults
 // or TestResult object using connector functions.
 type TestResultsOptions struct {
-	TaskID      string
-	TestName    string
-	Execution   *int
-	DisplayTask bool
+	TaskID        string
+	Execution     *int
+	DisplayTask   bool
+	FilterAndSort *TestResultsFilterAndSortOptions
+}
+
+// TestResultsFilterAndSortOptions holds all values required for filtering,
+// sorting, and paginating TestResult objects using connector functions.
+type TestResultsFilterAndSortOptions struct {
```

Comment:
Yeah it bothers me too, but I decided to be consistent with everything else in this file since all the options start with the service name (`BuildloggerOptions`, PerformanceOptions`, etc.) I can change that when I refactor this code.